src/types/User.ts

```diff
@@ -33,3 +35,5 @@ export interface User {
   anonMode: boolean;
   enteredVenueIds?: string[];
 }
+
+export type OnAvatarClick = (user: WithId<User>) => void;
```

Comment:
This is for `setSelectedProfile` logic. Will probably be removed, when we refactor that monster

Reply:
_nods_ I think I just meant that we should put it within that same component file, rather than in the generic types/User section, as it more relates to an implementation detail of that component.

Reply:
_nods_ I think I just meant that we should put it within that same component file, rather than in the generic types/User section, as it more relates to an implementation detail of that component.